packages/aws-cdk-lib/aws-ses/lib/configuration-set.ts

```diff
@@ -158,10 +167,23 @@ export class ConfigurationSet extends Resource implements IConfigurationSet {
       physicalName: props.configurationSetName,
     });
 
+    if (props.maxDeliveryDuration && !Token.isUnresolved(props.maxDeliveryDuration)) {
+      if (props.maxDeliveryDuration.toMilliseconds() < Duration.seconds(1).toMilliseconds()) {
+        throw new Error(`The maximum delivery duration must be greater than or equal to 5 minutes (300 seconds), got: ${props.maxDeliveryDuration.toMilliseconds()} milliseconds.`);
```

Comment:
Is this check needed if we already have the 5 min check?

Reply:
You're right. I removed the redundancy check.